src/components/ProgressBar/ProgressBar.js

```diff
@@ -0,0 +1,22 @@
+import styles from './ProgressBar.module.css';
+
+function ProgressBar(props) {
+  const { humidityPercentage, progressBarClass } = props ;
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.percentageNumber}>
+        <span>0</span>
+        <span>50</span>
+        <span>100</span>
+      </div>
+      <div className={`${styles.container} + ${progressBarClass}`}>
```

Comment:
you should use `classNames` lib, also what is ` + ` ???

`progressBarClass` -> `progressBarClassName`